assets/src/domain/shared/entities/tickets/helpers/getTicketStatusTextLabel.ts

```diff
@@ -0,0 +1,29 @@
+import { __ } from '@wordpress/i18n';
+
+import status from './status';
+import { Ticket } from '@edtrServices/apollo/types';
+import { TICKET_STATUS_ID } from '../constants';
+
+const getTicketStatusTextLabel = (ticket: Ticket): string => {
+	let ticketStatus = '';
+	switch (status(ticket)) {
+		case TICKET_STATUS_ID.SOLD_OUT:
+			ticketStatus = __('sold out', 'event_espresso');
+			break;
+		case TICKET_STATUS_ID.EXPIRED:
+			ticketStatus = __('expired', 'event_espresso');
+			break;
+		case TICKET_STATUS_ID.PENDING:
+			ticketStatus = __('pending', 'event_espresso');
+			break;
+		case TICKET_STATUS_ID.ONSALE:
+			ticketStatus = __('on sale', 'event_espresso');
+			break;
+		case TICKET_STATUS_ID.ARCHIVED:
+			ticketStatus = __('archived', 'event_espresso');
+			break;
```

Comment:
Few points here:
- Similar order of highest priority should be applied here.
- `TRASHED` instead of `ARCHIVED`
- No second argument to `__()`

Reply:
Updated, thx.